eclipse_development_process.adoc

```diff
@@ -406,23 +409,21 @@ _(Most of this section is borrowed and paraphrased from the excellent http://www
 
 Releases are, by definition, anything that is distributed outside of the committers of a project. If users are being directed to download a build, then that build has been released (modulo the exceptions below). All projects and committers must obey the Eclipse Foundation requirements on approving any release.
 
+A project can make official releases for one calendar year following a successful link:#6_3_5_Progress_Review[progress review] or link:#6_3_3_Release_Review[release review]. The project leadership chain may--at its discretion--require that the project engage in additional reviews (e.g. a progress or release review) prior to issuing a release.
```

Comment:
- do projects still need to send IP logs for approval for each release which is not a bug fix release ?
- do we have any standards which should/must be followed for each release, not regarding if there is a release review for the release or not, e.g. each release must have release notes

Reply:
@msohn 

_"do projects still need to send IP logs for approval for each release"_ -> No

This will be done only during a Release or Progress review. An IP log is not (never was) tight to a release. It was verified as part of the Release Review but technically an IP log represents the whole project.

_"do we have any standards which should/must be followed for each release"_ -> Yes

Yes. However, maybe we need some additional wording here. 

@waynebeaton I think technically the documentation requirement was captured by the Release Review definition in combination with _"All projects and committers must obey the Eclipse Foundation requirements on approving any release."_. Should there be some wording here that releases still require some documentation (I'm thinking Changelog, release record) and PMC approval?

Reply:
I generally try to avoid putting implementation details in the process document. IP Log reviews are how we meet the requirements of the IP Policy, which doesn't mention IP Logs specifically. That is, the IP Policy states "The Release Review is used as a final review to ensure that all necessary rights to the new Content have, in fact, been acquired."; the way that we do this is via an IP Log review.